examples/DefaultBoard/DefaultBoard.ino

```diff
@@ -41,6 +41,11 @@ void loop() {
       if(SDfileOpen) {
         // Write to the SD card, writes aux data
         writeDataToSDcard(board.sampleCounter);
+        if (!board.ledSDWrite)
+          board.activateLedSDWrite(true);
```

Comment:
move all this code to the `loop` function please. no new code in avoid new code in `DefaultBoard.ino`

Reply:
hmmm, not sure how to do that, SDfileOpen does not exist in "board" scope.  Ideas?